tests/vegalite/v5/test_api.py

```diff
@@ -730,7 +767,7 @@ def test_when_then_interactive() -> None:
         .encode(
             x="IMDB_Rating:Q",
             y="Rotten_Tomatoes_Rating:Q",
-            color=alt.when(predicate).then(alt.value("grey")),  # type: ignore[arg-type]
+            color=alt.when(predicate).then(alt.value("grey")),
```

Comment:
Not requiring this `# type: ignore` demonstrates that https://github.com/vega/altair/issues/3552 is fixed